demos/pi/pom.xml

```diff
@@ -16,4 +16,12 @@
     <version>3.2.0-SNAPSHOT</version>
   </parent>
 
+  <dependencies>
+    <dependency>
+      <groupId>it.unimi.dsi</groupId>
+      <artifactId>fastutil</artifactId>
```

Comment:
Without this dependency here (the same thing is also done in the twitter demo), we get
the following exception when we attempt to launch the application:

java.lang.NoClassDefFoundError: it/unimi/dsi/fastutil/objects/Object2IntLinkedOpenHashMap
	at com.datatorrent.lib.appdata.schemas.FieldsDescriptor.initialize(FieldsDescriptor.java:205)
	at com.datatorrent.lib.appdata.schemas.FieldsDescriptor.<init>(FieldsDescriptor.java:110)
	at com.datatorrent.lib.appdata.schemas.SnapshotSchema.initialize(SnapshotSchema.java:233)
	at com.datatorrent.lib.appdata.schemas.SnapshotSchema.<init>(SnapshotSchema.java:151)
	at com.datatorrent.lib.appdata.snapshot.AbstractAppDataSnapshotServer.setup(AbstractAppDataSnapshotServer.java:157)
	at com.datatorrent.lib.appdata.snapshot.AbstractAppDataSnapshotServer.setup(AbstractAppDataSnapshotServer.java:57)
	at com.datatorrent.stram.engine.Node.setup(Node.java:158)
	at com.datatorrent.stram.engine.StreamingContainer.setupNode(StreamingContainer.java:1283)
	at com.datatorrent.stram.engine.StreamingContainer.access$100(StreamingContainer.java:88)
	at com.datatorrent.stram.engine.StreamingContainer$2.run(StreamingContainer.java:1357)
Caused by: java.lang.ClassNotFoundException: it.unimi.dsi.fastutil.objects.Object2IntLinkedOpenHashMap
	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)